SPECS/bind/bind.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "bind-9.20.5.tar.xz": "19274fd739c023772b4212a0b6c201cf4364855fa7e6a7d3db49693f55db1ab8",
+  "bind-9.20.9.tar.xz": "3d26900ed9c9a859073ffea9b97e292c1248dad18279b17b05fcb23c3091f86d",
```

Comment:
@kgodara912, are you the owner of this PR? If so, could you also extend the summary with a link or other information pointing to where it's said that the CVE is fixed in version 9.20.9? I found it, but we want to streamline the review process and include all relevant information in the PR itself, so it's easier to understand why we've made specific patching decisions if we ever analyze a PR in the future.